test/passthrough_test.js

```diff
@@ -339,4 +339,21 @@ describe('passthrough requests', function(config) {
 
     xhr.send();
   });
+
+  it('allows a passthrough on an unhandledRequest via request.passthrough()', function(assert) {
```

Comment:
Can you add another test for return type of `request.passthrough()` is `originalXMLHttpRequest` isntance?

Reply:
Done. I actually should have looked back to the change I made all those months ago before I edited the Readme. It is the same thing that is returned from `createPassthrough`, which is actually a new instance of the native `XMLHttpRequest`. The new test and updated Readme accurately reflect that reality.